packages/dashboard/src/pages/Wallets/hooks/useCollectibles.ts

```diff
@@ -0,0 +1,53 @@
+import type { CollectibleProvider } from '../types'
+import {
+    createERC721Token,
+    ERC1155TokenAssetDetailed,
+    ERC721TokenAssetDetailed,
+    isSameAddress,
+    useChainId,
+} from '@dimensiondev/web3-shared'
+import type { AsyncStateRetry } from 'react-use/lib/useAsyncRetry'
+import { useAsyncRetry } from 'react-use'
+import { PluginServices } from '../../../API'
+import { uniqWith } from 'lodash-es'
+
+export function useCollectibles(
```

Comment:
I think that should in the web3-shared package.

Reply:
I don't think that should be a move to web3-shared. Because here need to fetch data from third-party API.

```
const result = await PluginServices.Wallet.getAssetsListNFT(address.toLowerCase(), chainId, provider, page, 20)
```

the `page` param will be hard pass to hook if that moves to `web3-shared`.